chat.py

```diff
@@ -46,11 +53,12 @@
             "ban" : {
                 "words" : [],
                 "ip" : [],
-                "length": 2147483647
+                "length" : 2147483647
             },
             "ENTER_AFTER_PROMISE" : False,
             "SHOW_ENTER_MESSAGE" : False,
-            "AUTO_REMOVE_OFFLINE" : False
+            "AUTO_REMOVE_OFFLINE" : False,
+            "FLUSH_INTERVAL" : 60
```

Comment:
```suggestion
            "FLUSH_INTERVAL" : 30
```

@035966-L3 感觉 60 有点慢，30s 应该可以